packages/core/src/providers/ping-id.js

```diff
@@ -0,0 +1,34 @@
+/** @type {import(".").OAuthProvider} */
```

Comment:
Could we convert this to TypeScript?

Also, make sure to document the code via JSDoc comments, have a look at https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/github.ts for a good example.

Reply:
Sure, I'll convert it into typescript and also document the code via JSDoc

Reply:
Yes, it has been converted to TypeScript